modules/Bio/EnsEMBL/Compara/PipeConfig/UpdateReferenceDatabase_conf.pm

```diff
@@ -255,6 +281,7 @@ sub core_pipeline_analyses {
                 'compara_db'            => '#ref_db#',
                 'allowed_species_file'  => $self->o('allowed_species_file'),
                 'xml_file'              => $self->o('xml_file'),
+                'members_dumps_dir'     => '#member_dumps_dir#',
```

Comment:
We need to make sure this line disappears before merging (I guess it is here because of the conflicts we have talked about before).